### ISSUE TITLE
Non-utf8 binary crashes Morphix.atomorphiform

### ISSUE THREAD

**Issue by methyl** [philosodad/morphix#24]
In Elixir you can have a map where keys are non-utf8 binary like this
```
%{<<255>> => 123}
```

Calling `Morphix.atomorphiform` crashes with following error:
```
%{<<255>> => 123} |> Morphix.atomorphiform
** (ArgumentError) argument error
    :erlang.binary_to_atom(<<255>>, :utf8)
    (morphix) lib/morphix.ex:534: Morphix.atomize_binary/2
    (morphix) lib/morphix.ex:517: anonymous fn/4 in Morphix.morphiform!/3
    (stdlib) maps.erl:257: :maps.fold_1/3
    (morphix) lib/morphix.ex:331: Morphix.atomorphiform/1
```

I think there are two different behaviors that would work better in this case. Either leave the binary unchanged, making `%{<<255>> => 123} |> Morphix.atomorphiform` a no-op (as you already do with numbers for example) or return an error tuple.

**Comment by philosodad** [comment]
Atomorphify or Atomorphiform with the `:safe` argument will implement the second behavior, so for consistency the fix will just ignore non-printable binaries.

**Comment by philosodad** [comment]
Fixed in version 0.8.1
